stdlib/websub/src/main/ballerina/src/websub/subscriber_service_endpoint.bal

```diff
@@ -61,12 +92,18 @@ public type Listener object {
         return ();
     }
 
+# Forcefully stops the `websub:Listener`.
+# ```ballerina
+# error? result = websubListener.__immediateStop();
+# ```
+#
+# + return - () or else an `error` upon failure to stop the listener
```

Comment:
```suggestion
# + return - () or else an `error` upon failure to stop the listener
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.